website/docs/reference/widgets/table/README.md

```diff
@@ -28,7 +28,7 @@ These properties allow you to edit the table itself. All these properties are pr
 | **Default Selected Row**       | Sets the default selected row of the table. It takes the index no. of the row as an input.                                                                                                              |
 | **Default Row Height**         | Sets the height of the row in the table - Default, short or tall.                                                                                                                                       |
 | **Enable multi-row selection** | Allows multiple rows of a table to be selected. The rows are populated in the selectedRows field.                                                                                                       |
-| **Table Record Count**         | The user has to input the total number of records returned as part of a query or API call, which will be displayed in a table. It is only visible when you choose to enable **server-side pagination**. |
+| **Total Record Count**         | The user has to input the total number of records returned as part of a query or API call, which will be displayed in a table. It is only visible when you choose to enable **server-side pagination**. |
```

Comment:
Here in the description I think we should make it clear that the total record count is used to control the next page control when using server side pagination.  so you can rephrase it as 

The user can provide the total number of rows returned by a query or API, which helps in calculating the total number of pages on the table when using server-side pagination.